PublicationsRouter/oai-pmh/RIOXX_XWALK.md

```diff
@@ -0,0 +1,88 @@
+	* {1..n} – one or more (i.e. a mandatory element)
+* Column 2 - PubRouter internal metadata JSON fields from which the output is derived
+* Column 3 - the output XML element construction (format) - see note. 
+
+**NOTE: XML Format column** - Field holders are shown in `[square brackets]`, in the output XML these field holders are replaced by data from the indicated PubRouter JSON metadata fields.  For example, `[journal.title]` would be replaced by the actual title of the journal.  Any other text is output as it appears in the format.
+
+| rioxx terms | PubRouter Metadata (source) | XML Format |
+|:-----------------------------|:-------------------------|:------------------------------------------------------------|
+| [ali:license_ref](http://www.rioxx.net/schema/v2.0/rioxx/ali_1_0.html#license_ref) <br>{0..n} | license_ref.url <br> license_ref.url.start  | `<ali:license_ref start=”[license_ref.url.start]”> [license_ref.url] </ali:license_ref>` |
+| [ali:free_to_read](http://www.rioxx.net/schema/v2.0/rioxx/ali_1_0.html#free_to_read) <br>{0..1} | free2read.start <br> free2read.end | `<ali:free_to_read start_date=[free2read.start] end_date=[free2read.end]></ali:free_to_read> |
+| [dc:description](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__description)  <br>{0..n} | provider_agent | `<dc:description>From [provider_agent] via Jisc Publications Router.</dc:description>` |
+| [dc:description](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__description)  <br>{0..n}  | publication_status | `<dc:description>Publication status: [publication_status]</dc:description>` | 
+| [dc:description](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__description)  <br>{0..n}  | history_date.date_type <br> history_date.date | `<dc:description>History: [history_date.date_type], [history_date.date] </dc:description>` |
+| [dc:identifier](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__identifier)  <br>{0..1} | links.url <br> This field MUST contain an HTTP URI which is a persistent identifier for the resource. | `<dc:identifier>[links.url]</dc:identifier>` |
+| [dc:language](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__language) <br>{0..n} | article.language | `<dc:language>[article.language]</dc:language>` |
+| [dc:publisher](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__publisher)  <br>{0..1} | journal.publisher | `<dc:publisher>[journal.publisher]</dc:publisher>` |
+| [dc:source](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__source)  <br>{0..n} | journal.identifier.type <br> journal.identifier.id  | `<dc:source>[journal.identifier.type]: [journal.identifier.id]</dc:source>` |
+| [dc:source](http://www.rioxx.net/schema/v2.0/rioxx/terms_.html#http___purl.org_dc_terms__source)  <br>{0..1} | journal.title | `<dc:source>[journal.title]</dc:source>` |
```

Comment:
There is no need for PubRouter to comply with RIOXX as Router is not a repository, the only requirement is to populate the available metadata in a RIOXX format.

The output XML will have as many `dc:source` elements as rows are defined in this table, therefore I think it is better to have them in individual rows

Reply:
See earlier comments.

Also, it does not make sense for one of these rows to be {0..n} and the other {0..1}, since in the end they are both dc:source elements.